pkgs/tools/audio/beets/plugins/filetote.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  beets,
+  beetsPackages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "beets-filetote";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "gtronset";
+    repo = "beets-filetote";
+    rev = "v${version}";
+    hash = "sha256-q+CWn8p7/16Z6mAzOfh0dqg96CO7X1u/jHPuhK61bVw=";
+  };
+
+  postPatch = ''
+    sed -i -e '/install_requires/,/\]/{/beets/d}' setup.py
+    sed -i -e '/namespace_packages/d' setup.py
+    printf 'from pkgutil import extend_path\n__path__ = extend_path(__path__, __name__)\n' >beetsplug/__init__.py
+
+    # beets v2.1.0 compat
+    # <https://github.com/beetbox/beets/commit/0e87389994a9969fa0930ffaa607609d02e286a8>
+    sed -i -e 's/util\.py3_path/os.fsdecode/g' tests/_common.py
```

Comment:
This also does nothing, as the tests have been updated to work with latest beets.